test/OrchardCore.Tests/Serializers/JsonSerializerTests.cs

```diff
@@ -72,10 +73,10 @@ await context.UsingTenantScopeAsync(async scope =>
 
             var session = scope.ServiceProvider.GetRequiredService<YesSql.ISession>();
 
-            await session.SaveAsync(newUser);
-            await session.SaveChangesAsync();
+            await session.SaveAsync(newUser, false, null, CancellationToken.None);
```

Comment:
Instead of passing `false` and `null` its probably better to just use `cancellationToken: CancellationToken.None` paramters instead. so in this case it would look like this

> await session.SaveAsync(newUser, cancellationToken: CancellationToken.None);

Apply this everywhere you added `false` and `null` arguments in this PR.